streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -264,7 +265,10 @@ public void shouldAttemptToDeleteStateDirectoryWhenCloseDirtyAndEosEnabled() thr
         ctrl.replay();
 
         task = createStatefulTask(createConfig(true, "100"), true, stateManager);
-        task.transitionTo(Task.State.CLOSING);
+        task.transitionTo(Task.State.RESTORING);
+        task.transitionTo(Task.State.RUNNING);
+        task.transitionTo(Task.State.SUSPENDED);
```

Comment:
Why we have to transit to SUSPENDED before prepare-closing? Originally we want to check that CREATED state can still trigger close.

Reply:
Ack. We can remove this.